inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -0,0 +1,145 @@
+                of the wind tunnel in the [x, y, z] axes, in meters. This is a
+                natural description with the default OpenFOAM simulator.
+        """
+
+        self.object_path = object_path
+        self.flow_velocity = flow_velocity
+
+        if domain is None:
+            logging.info("Using a default domain: [[-5, 15], [-4, 4], [0, 8]].")
+            self.domain = [[-5, 15], [-4, 4], [0, 8]]
+
+    def simulate(self,
+                 simulator: Simulator = OpenFOAM(),
+                 output_dir: Optional[Path] = None,
+                 simulation_time: float = 100,
+                 write_interval: float = 50,
+                 n_cores: int = 1):
+        """Simulates the wind tunnel scenario.
+        
+        Args:
+            simulator: Simulator to use for the simulation.
+            output_dir: Path to the directory where the simulation output
+                is downloaded.
+            simulation_time: Simulation time, in seconds.
+            write_interval: Interval between simulation outputs, in seconds.
+            n_cores: Number of cores to use for the simulation.
+            """
+
+        self.simulation_time = simulation_time
+        self.write_interval = write_interval
+        self.n_cores = n_cores
+
+        output_path = super().simulate(
+            simulator,
+            output_dir=output_dir,
+            n_cores=n_cores,
+            openfoam_solver="simpleFoam",
+        )
+
+        return output_path
+
+    @singledispatchmethod
+    @classmethod
+    def get_config_filename(cls, simulator: Simulator):  # pylint: disable=unused-argument
+        raise ValueError(
+            f"Simulator not supported for `{cls.__name__}` scenario.")
+
+    @singledispatchmethod
+    def gen_aux_files(self, simulator: Simulator, input_dir: str):
+        raise ValueError(
+            f"Simulator not supported for `{self.__class__.__name__}` scenario."
+        )
+
+    @singledispatchmethod
+    def gen_config(self, simulator: Simulator, input_dir: str):
+        raise ValueError(
+            f"Simulator not supported for `{self.__class__.__name__}` scenario."
+        )
+
+
+@WindTunnel.get_config_filename.register
+def _(self, simulator: OpenFOAM):  # pylint: disable=unused-argument
+    pass
+
+
+@WindTunnel.gen_aux_files.register
+def _(self, simulator: OpenFOAM, input_dir):  # pylint: disable=unused-argument
+    # The WindTunnel with OpenFOAM requires changing multiple files
+    template_dir = os.path.join(os.path.dirname(__file__),
+                                OPENFOAM_TEMPLATE_INPUT_DIR)
+
+    # Copy all files from the template dir to the input directory
+    for directory in os.listdir(template_dir):
+        shutil.copytree(os.path.join(template_dir, directory),
+                        os.path.join(input_dir, directory))
+
+    # Remove all files that have .jinja in input_dir
+    remove_files_with_tag(input_dir, ".jinja")
```

Comment:
I would avoid copying the files with `.jinja` extension in the first place, to avoid having this function.